pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -1,46 +0,0 @@
-{ lib
-, stdenv
-, fetchpatch
-, kernel
-, commitDate ? "2023-06-28"
-# bcachefs-tools stores the expected-revision in:
-#   https://evilpiepirate.org/git/bcachefs-tools.git/tree/.bcachefs_revision
-# but this does not means that it'll be the latest-compatible revision
-, currentCommit ? "4d2faeb4fb58c389dc9f76b8d5ae991ef4497e04"
-, diffHash ? "sha256-DtMc8P4lTRzvS6PVvD7WtWEPsfnxIXSpqMsKKWs+edI="
-, kernelPatches # must always be defined in bcachefs' all-packages.nix entry because it's also a top-level attribute supplied by callPackage
-, argsOverride ? {}
-, ...
-} @ args:
-# NOTE: bcachefs-tools should be updated simultaneously to preserve compatibility
-(kernel.override ( args // {
-
-  argsOverride = {
-    version = "${kernel.version}-bcachefs-unstable-${commitDate}";
-    modDirVersion = kernel.modDirVersion;
-
-    extraMeta = {
-      homepage = "https://bcachefs.org/";
-      branch = "master";
-      maintainers = with lib.maintainers; [ davidak Madouura pedrohlc raitobezarius YellowOnion ];
-    };
-  } // argsOverride;
-
-  structuredExtraConfig = with lib.kernel; {
-    BCACHEFS_FS = module;
-    BCACHEFS_QUOTA = option yes;
-    BCACHEFS_POSIX_ACL = option yes;
```

Comment:
Is it safe to drop `BCACHEFS_QUOTA`/`BCACHEFS_POSIX_ACL` here? Should these move to common-config with `whenAtLeast 6.7`?

Reply:
I'm not sure, but this might be best left to the user.

Reply:
On the other hand, we're changing the kernel for people's bcachefs installations...
Maybe a warning would be better?

Reply:
I think these fall under sane defaults (similar quota/acl settings are set for other filesystems in the common config file) and the default should be to set them -- anyone who doesn't want posix acls or quotas (for whatever reason) is free to override it if they want. It's left up to the user in both cases.

Reply:
If you feel strongly that these should not be explicitly added there, then I'd agree that the warning (and backport) is required, there's a good chance this can brick an install.